components/menu/menu-item-link.js

```diff
@@ -129,19 +136,27 @@ class MenuItemLink extends (newWindowIconEnabled ? LinkMixin(MenuItemMixin(LitEl
 		return null;
 	}
 
+	// remove this function when cleaning up GAUD-8369-menu-item-link-click-changes
 	_onClick() {
 		if (this.shadowRoot) this.shadowRoot.querySelector('a').dispatchEvent(new CustomEvent('click'));
```

Comment:
This line was added for a [VoiceOver bug](https://github.com/BrightspaceUI/core/pull/407) that seems to be fixed. I'll call that out in the test notes.